conda_build/metadata.py

```diff
@@ -121,25 +122,36 @@ def get_selectors(config: Config) -> dict[str, bool]:
     # Remember to update the docs of any of this changes
     plat = config.host_subdir
     d = dict(
-        linux=plat.startswith("linux-"),
         linux32=bool(plat == "linux-32"),
         linux64=bool(plat == "linux-64"),
-        emscripten=plat.startswith("emscripten-"),
-        wasi=plat.startswith("wasi-"),
-        arm=plat.startswith("linux-arm"),
-        osx=plat.startswith("osx-"),
+        arm=plat.startswith(("linux-arm", "osx-arm", "win-arm")),
```

Comment:
Let's go with the following?
```suggestion
        arm=plat.startswith("linux-arm"),
```
`arm` was true for `linux-armv7l` platform and not true for `linux-aarch64` platform. Since `osx-arm64` and `win-arm64` are similar to `linux-aarch64`, let's not add them.

Reply:
In other words `arm` is only for `32-bit` architectures.